libraries/ESP8266WebServer/examples/HelloServer/HelloServer.ino

```diff
@@ -67,6 +67,17 @@ void setup(void) {
     server.send(200, "text/plain", "this works as well");
   });
 
+  server.on("/gif", []() {
+    uint8_t gif[] = {
```

Comment:
This should be a static at least. Or be moved out to global space.

Reply:
Done.